### PR TITLE
Move website to SvelteKit

### PR DESCRIPTION
**Pull request by Narigo**
We're moving to SvelteKit - page was presented today in the Orga Meeting

### CODE REVIEW

**Comment by meldron** [comment]
Looks good to me!

File: package.json [hunk 1]
```diff
@@ -1,63 +1,42 @@
...
+	"private": true,
+	"engines": {
+		"node": ">= 18.0.0"
+	},
+	"scripts": {
+		"dev": "vite dev",
+		"build": "vite build",
+		"preview": "vite preview",
+		"test": "pnpm run test:unit",
+		"test:e2e": "playwright test",
```
Comment (lowsky) [inline package.json:13]: If there are no real ui e2e tests, I suggest to remove that and the additional playwright dependencies.
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -1,63 +1,42 @@
...
+		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
+		"test:unit": "vitest",
+		"lint": "prettier --plugin-search-dir . --check . && eslint .",
+		"format": "prettier --plugin-search-dir . --write ."
+	},
+	"packageManager": "pnpm@8.1.0",
+	"devDependencies": {
+		"@fontsource/poppins": "4.5.10",
+		"@fontsource/source-sans-pro": "4.5.11",
+		"@playwright/test": "1.32.1",
```
Comment (lowsky) [inline package.json:24]: ' having unit tests is perfect, and I love you added vitest!! - but playwrite is not really needed, 
I suggest to remove it:
```suggestion
```

File: playwright.config.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import type { PlaywrightTestConfig } from '@playwright/test';
```
Comment (lowsky) [inline playwright.config.ts:1]: This seems to be the default template, but as there are no real e2e tests - hey it is a static page  - I suggest to remove it :-)

File: test/participants.js [hunk 4]
```diff
@@ -1,166 +0,0 @@
-const assert = require("assert")
```
Comment (lowsky) [inline test/participants.js:1]: this was helpful to test the user's contributions. Why has it been removed? Or has it been moved? I don't see it...

**Review by lowsky — COMMENTED** [review]
Big Thanks! - Always great to use the modernisation and cleanup.

Looks great to me !

There are just a tiny suggestion, please have a look.

**Review by lowsky — APPROVED** [review]

**Comment by Narigo** [comment]
Thanks, I've rebased it again after the change to the `package.json` and fixed a small error in the participants tests

**Comment by Narigo** [comment]
And Playwright is removed now as suggested! :) Thanks @lowsky, we'll merge then!

**Pull request merged**